schoolopy/main.py

```diff
@@ -2003,14 +2085,29 @@ def get_message(self, message_id):
 
     # Implement search, resource collections, resource templates
 
+    def _like_item(self, pathItems, unlike=False):
```

Comment:
I'm on the fence about having a separate helper function for liking things, but if liking is going to be really complex then it does make sense. However I don't love the use of `unlike=False`, which creates some degree of asymmetry between `like` and `unlike`. I think if we're going to make a separate method it might be better to have a non-optional parameter `like` and pass a boolean for whether we will like the object.

Reply:
This is just a private method for allowing other methods (like, unlike) to work with it, so I don't have to copy paste in the comment like and comment unlike methods.